stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messages/QuotedMessage.kt

```diff
@@ -37,43 +41,74 @@ import io.getstream.chat.android.compose.ui.theme.ChatTheme
  * @param message Message to show.
  * @param modifier Modifier for styling.
  * @param onLongItemClick Handler when the item is long clicked.
+ * @param onQuotedMessageClick Handler for quoted message click action.
  */
+@OptIn(ExperimentalFoundationApi::class)
 @Composable
 public fun QuotedMessage(
     message: Message,
     modifier: Modifier = Modifier,
     onLongItemClick: (Message) -> Unit,
+    onQuotedMessageClick: (Message) -> Unit,
 ) {
     val user = message.user
+    val isMyMessage = message.isMine()
 
-    Row(modifier = modifier, verticalAlignment = Alignment.Bottom) {
-        Avatar(
-            modifier = Modifier.size(24.dp),
-            imageUrl = user.image,
-            initials = user.initials,
-            textStyle = ChatTheme.typography.captionBold,
-        )
+    val messageBubbleShape = if (isMyMessage) ChatTheme.shapes.myMessageBubble else ChatTheme.shapes.otherMessageBubble
+
+    Row(
+        modifier = modifier.combinedClickable(
+            interactionSource = MutableInteractionSource(),
+            indication = null,
+            onLongClick = { onLongItemClick(message) },
+            onClick = { onQuotedMessageClick(message) }
+        ),
+        verticalAlignment = Alignment.Bottom
+    ) {
+        if (!isMyMessage) {
+            Avatar(
+                modifier = Modifier
+                    .padding(start = 2.dp)
+                    .size(24.dp),
+                imageUrl = user.image,
+                initials = user.initials,
+                textStyle = ChatTheme.typography.captionBold,
+            )
 
-        Spacer(modifier = Modifier.size(8.dp))
+            Spacer(modifier = Modifier.size(8.dp))
+        }
 
         MessageBubble(
-            shape = ChatTheme.shapes.otherMessageBubble, color = ChatTheme.colors.barsBackground,
+            modifier = Modifier.weight(1f, fill = false),
+            shape = messageBubbleShape, color = ChatTheme.colors.barsBackground,
             content = {
-                Column {
-                    MessageAttachmentsContent(
-                        message = message,
-                        onLongItemClick = {}
-                    )
-
-                    if (message.text.isNotEmpty()) {
-                        MessageText(
-                            isQuote = true,
+                Row {
+                    if (message.attachments.isNotEmpty()) {
+                        MessageAttachmentsContent(
                             message = message,
-                            onLongItemClick = onLongItemClick
+                            onLongItemClick = {},
+                            isQuote = true
```

Comment:
As mentioned, if we had the QuotedMessageAttachmentsContent composable, we can just use it here w/o changing the default behavior of the previous component